docs/_posts/guides/9999-01-02-browsers.md

```diff
@@ -80,6 +80,10 @@ spark.once(`ready`, function() {
 
 By setting `spark.config.credentials.clientType` to `confidential`, `initiateLogin` will know to ask for an auth code instead of an access token.
 
+A `state` parameter is used to verify that the response from grant flow has not been tampered with along the way. It is recommended that you set this to a value that is verifiable once the user gives permission and the web browser is sent to your `redirect_uri`. A second use for this parameter is to encode basic state information like an internal user ID or the URL of the last page they were on before entering the grant flow.
+
+Include a `state` parameter in the `authorizationString`.
+
```

Comment:
Should we describe how the `state` comes back and can be used if developer decides to populate it?